src/js/profile/subcategory.js

```diff
@@ -49,16 +49,27 @@ export class Subcategory extends Observable {
         let formattingConfig = this.formattingConfig;
         let index = 0;
         let lastIndex = Object.entries(detail.indicators).length - 1;
-
-        for (const [title, indicatorData] of Object.entries(detail.indicators)) {
-            let isLast = index === lastIndex;
-            let i = new Indicator(formattingConfig, wrapper, title, indicatorData, detail, isLast);
-            this.bubbleEvents(i, [
-                'profile.chart.saveAsPng', 'profile.chart.valueTypeChanged',
-                'profile.chart.download_csv', 'profile.chart.download_excel', 'profile.chart.download_json', 'profile.chart.download_kml',
-                'point_tray.subindicator_filter.filter'
-            ]);
-            index++;
+        let isEmpty = JSON.stringify(detail.indicators) === JSON.stringify({});
```

Comment:
@emre2038 we need to revisit that and make the checks a little better and more meaningful.